src/opds_dumper.cpp

```diff
@@ -99,12 +100,12 @@ std::string fullEntryXML(const Book& book, const std::string& rootLocation, cons
     return render_template(RESOURCE::templates::catalog_v2_entry_xml, data);
 }
 
-std::string partialEntryXML(const Book& book, const std::string& rootLocation)
+std::string partialEntryXML(const ServerConfiguration& configuration, const Book& book)
```

Comment:
I don't agree with this change.

Cons:
- access to more data than needed by this function

Pros:
- ???

Reply:
I made this for consistency.
The idea is that those set of free functions to generate xml are black box function generating the xml in accordance with the ServerConfiguration, what ever is actually used.

I have no strong opinion on this,except that passing the actual needed data may leads to too much parameters.
And if several functions are using a ServerConfiguration I prefer having all functions using them.